docs/reference/contributing/target/target.md

```diff
@@ -19,7 +19,7 @@ cd ..
 
 #### Target description
 
-Add the target description to `mbed-os\targets\targets.json`:
+Add the target description to `mbed-os\targets\targets.json` using keys described in the [adding and configuring targets section](https://os.mbed.com/docs/v5.6/tools/adding-and-configuring-targets.html). We recommend that you run the [targets lint script](https://os.mbed.com/docs/v5.6/tools/adding-and-configuring-targets.html#style-guide) on your target hierarchy before submitting your pull request:
```

Comment:
- Please change to "...using keys that the Adding and configuring targets section describes. We recommend..."
- Please change the first link to:
```<a href="https://os.mbed.com/docs/v5.6/tools/adding-and-configuring-targets.html" target="_blank">Adding and configuring targets section</a>```
- Please change the second link to:
```<a href="https://os.mbed.com/docs/v5.6/tools/adding-and-configuring-targets.html#style-guide" target="_blank">targets lint script</a>```